test/api/request/request.test.js

```diff
@@ -421,11 +421,26 @@ describe('#Request', () => {
             Valentine: 'sister'
           },
           year: '5270',
-          defeatedBugsAt: 11
+          defeatedBugsAt: 11,
+          relations: {
+            'lebron': ['james', 'curry', 'harden'],
+            'kobe': ['bryant', 'jordan', 'love']
+          },
+          powers: {
+            fire: {
+              mana: 10,
+              damage: 100
+            }
+          }
         };
       });
 
       describe('#getBodyArray', () => {
+        it('extracts the required parameter (lodash request)', () => {
+          should(request.getBodyArray('relations.lebron'))
+            .exactly(request.input.body.relations.lebron);
```

Comment:
`_.get` path accept indexes for arrays like "relations.lebron[1]", could be nice to add a test case.